src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="wrap" data-qa="big">
+      <img
+        class="img"
+        src="images/Search.svg"
+        alt="a img"
+      >
+      <input
+        type="search"
+        class="input"
+        data-qa="keypress"
+        placeholder="Try &#x22;Los Angeles&#x22;"
+      >
+    </form>
+    <form class="wrap" data-qa="small">
+      <img
+        class="img_sm"
```

Comment:
```suggestion
        class="img_small"
```
It's not clear what 'img_sm' could mean. Let's choose better names